src/services/mongoose/schema.ts

```diff
@@ -16,13 +16,13 @@ import { WalletRecord } from "./wallets"
 
 const Schema = mongoose.Schema
 
-const dbMetadataSchema = new Schema({
+const dbMetadataSchema = new Schema<DbMetadataRecord>({
   routingFeeLastEntry: Date, // TODO: rename to routingRevenueLastEntry
 })
 export const DbMetadata = mongoose.model("DbMetadata", dbMetadataSchema)
 
 const walletInvoiceSchema = new Schema<WalletInvoiceRecord>({
-  _id: String, // hash of invoice
+  _id: { type: String }, // hash of invoice
```

Comment:
I think by mongoose convention _id should be `ObjectId` not `String`. but maybe that is the Type, not what is saved in mongodb? not sure but something that should be verified. 

https://mongoosejs.com/docs/guide.html#_id

Reply:
wallet invoice should be a string because is a custom id (payment hash)